ManualPrinting.c

```diff
@@ -0,0 +1,14 @@
+#include "main.h"
+#include <stdio.h>
+
+void PrintWiringManual(void)
+{
+  int PairNumber;
+  for(PairNumber=1;PairNumber<=25;PairNumber++)
```

Comment:
Do not hardcode numbers like 25. It makes it hard to maintain when adding or removing colors.